lib/defaults/defaults.go

```diff
@@ -1066,6 +1066,10 @@ var (
 	// TransientErrorTimeout specifies the maximum amount of time to attempt
 	// an operation experiencing transient errors
 	TransientErrorTimeout = 15 * time.Minute
+
+	// WormholeImg is the docker image reference to use when embedding wormhole
+	// Note: This is a build parameter, and the build scripts will replace this with an image reference
+	WormholeImg = "<build param>"
```

Comment:
Maybe add some kind of init-time assertion that this has been replaced during build time, e.g. something like:

```go
func init() {
  if WormholeImg == "<build param>" {
    panic("defaults.WormholeImg wasn't set during build time. Did you build gravity using Makefile?")
  }
}
```

Reply:
it's a good point, but would this trigger when running things like test's, where we may not care if the parameter has been set...